src/main/java/com/library/model/entity/Book.java

```diff
@@ -0,0 +1,43 @@
+package com.library.model.entity;
+
+import jakarta.persistence.Column;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.FetchType;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.JoinTable;
+import jakarta.persistence.ManyToMany;
+import jakarta.persistence.ManyToOne;
+import java.util.List;
+
+@Entity
+@Getter
+@Setter
+@NoArgsConstructor
+public class Book extends ParentEntity {
+
+    @Column
+    private String title;
+
+    @Column
+    private Short publicationYear;
```

Comment:
How is this column named in DB? And why were ripDay (...Date) and birthDay (...Date) renamed in Author?

Reply:
publicationYear in DB is named publication_year... In Author i rename their, because new names are shorter than from DB

Reply:
ok